samples/drivers/drv2605/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_LOG=y
+CONFIG_HAPTICS=y
+CONFIG_HAPTICS_DRV2605=y
```

Comment:
The device driver ~should select~ already selects the driver automatically using `depends on DT_HAS_TI_DRV2605_ENABLED` and `default y` making this selection redundant :)

Reply:
ack